src/test/DepositWithdraw.t.sol

```diff
@@ -553,8 +643,49 @@ contract DepositWithdrawTests is EigenLayerTestHelper {
                 )
             );
 
-        _testDepositToStrategy(sender, amountToDeposit, underlyingToken, stethStrategy);
+        // REMAINDER OF CODE ADAPTED FROM `_testDepositToStrategy`
+        // _testDepositToStrategy(sender, amountToDeposit, underlyingToken, stethStrategy);
+
+        // whitelist the strategy for deposit, in case it wasn't before
+        {
+            cheats.startPrank(strategyManager.strategyWhitelister());
+            IStrategy[] memory _strategy = new IStrategy[](1);
+            _strategy[0] = stethStrategy;
+            strategyManager.addStrategiesToDepositWhitelist(_strategy);
+            cheats.stopPrank();
+        }
+
+        uint256 operatorSharesBefore = strategyManager.stakerStrategyShares(address(this), stethStrategy);
+        // check the expected output
+        uint256 expectedSharesOut = stethStrategy.underlyingToShares(amountToDeposit);
+
+        underlyingToken.transfer(address(this), amountToDeposit);
+        cheats.startPrank(address(this));
+        underlyingToken.approve(address(strategyManager), type(uint256).max);
+        strategyManager.depositIntoStrategy(stethStrategy, underlyingToken, amountToDeposit);
+
+        //check if depositor has never used this strat, that it is added correctly to stakerStrategyList array.
+        if (operatorSharesBefore == 0) {
+            // check that strategy is appropriately added to dynamic array of all of sender's strategies
+            assertTrue(
+                strategyManager.stakerStrategyList(address(this), strategyManager.stakerStrategyListLength(address(this)) - 1)
+                    == stethStrategy,
+                "_testDepositToStrategy: stakerStrategyList array updated incorrectly"
+            );
+        }
+            
+        // check that the shares out match the expected amount out
+        // the actual transfer in will be lower by 1 wei than expected due to stETH's internal rounding
+        // to account for this we check approximate rather than strict equivalence here
+        {
+            uint256 actualSharesOut = strategyManager.stakerStrategyShares(address(this), stethStrategy) - operatorSharesBefore;
+            require((actualSharesOut * 1000) / expectedSharesOut > 998, "too few shares");
```

Comment:
shouldn't this slippage be way less?

Reply:
re-check with the changes in https://github.com/Layr-Labs/eigenlayer-contracts/pull/36/commits/e8cc3f9979071b22c6b121500abfc6e17faa4443